### PR TITLE
:sparkles: Add counts and styling for incident / issues 

### PR DESCRIPTION
**Pull request by ibolton336**
Screenshot 2024-12-09 at 4 43 25 PM

### CODE REVIEW

File: webview-ui/src/components/ViolationsCount/ViolationsCount.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  violationsCount: number;
+  incidentsCount: number;
+}
+
+export const ViolationsCount: React.FC<ViolationsCountProps> = ({
+  violationsCount,
+  incidentsCount,
+}) => {
+  const getStatusText = (count: number) => {
+    if (count === 0) return "(No incidents found)";
```
Comment (rszwajko) [inline webview-ui/src/components/ViolationsCount/ViolationsCount.tsx:16]: add `{}`  for the if-block - it triggers multiple warnings on GH

File: webview-ui/src/components/ViolationsCount/ViolationsCount.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    if (count === 1) return "(1 incident found)";
```
Comment (rszwajko) [inline webview-ui/src/components/ViolationsCount/ViolationsCount.tsx:17]: add `{}`  for the if-block - it triggers multiple warnings on GH

**Review by rszwajko — APPROVED** [review]
Looks good - see comments for minor issues

**Review by sjd78 — APPROVED** [review]
If you'd like to remove `const [analysisMessage, setAnalysisMessage] = useState("");` in `AnalysisPage.tsx` since it isn't set anywhere and it just throwing lint errors, that would be sweet.

Else, LGTM

**Review by rszwajko — APPROVED** [review]

**Pull request merged**